pkg/util/test_logger.go

```diff
@@ -29,13 +29,12 @@ func TestFlowLogger(t require.TestingT) *logging.Logger {
 		t.Helper()
 	}
 
-	sink, err := logging.WriterSink(os.Stderr, logging.SinkOptions{
+	l, err := logging.New(os.Stderr, logging.Options{
```

Comment:
TestFlowLogger isn't needed anymore since the interface is back to log.Logger; this can be removed and all instances of using it can be reverted back to `util.TestLogger(t)`.